### DISCUSSION TITLE
Validate Code into the August Account Info - No where to input

### DISCUSSION THREAD

**Discussion by k-hud** [homebridge-plugins/homebridge-august#64] [Q&A]
This step isn't very clear: "Input your Validate Code into the August Account Info"

There's no where to input this code once you receive it from August. 

The only fields on the "August Account Info" in the plugin configuration screen are: 
- Username
- Password
- Notice (which is prefilled with some notification info)

Where does the validation code go? I've restarted Hoobs and nothing changes with the plugin fields available.

**Comment by marquesed** [comment]
+1 was not able to see where to input the validation code.

**Comment by donavanbecker** [comment]
This plugin isn't developed for HOOBs.

I would recommend[ moving to homebridge](https://github.com/homebridge/homebridge-raspbian-image/wiki/How-To-Migrate-From-HOOBS-To-Homebridge).
